models/utilities_test.go

```diff
@@ -35,28 +35,26 @@ func TestWeCanGetTheParentIDsFromAListOfComments(t *testing.T) {
 		t.Error("The default output of the parent ID getter should be \"0\"")
 	}
 
-	CreateUser("person@example.com", "friendlyPerson", "password")
-	person := User{}
-	person.FindByEmail("person@example.com")
+	person, _ := UserFactory("person@example.com", "friendlyPerson", "password")
 	// create a post and a weird comment thread
 	post, _ := person.CreatePost("Post Title", "post-title", "lorem ipsum", "image")
-	topLevelComment1, _ := person.CommentOnPost(post, nil, "text", "text", "")
-	topLevelComment2, _ := person.CommentOnPost(post, nil, "text", "text", "")
-	topLevelComment3, _ := person.CommentOnPost(post, nil, "text", "text", "")
-	topLevelComment4, _ := person.CommentOnPost(post, nil, "text", "text", "")
-	topLevelComment5, _ := person.CommentOnPost(post, nil, "text", "text", "")
+	topLevelComment1, _ := person.CommentOnPost(post, nil, "text", "text")
+	topLevelComment2, _ := person.CommentOnPost(post, nil, "text", "text")
+	topLevelComment3, _ := person.CommentOnPost(post, nil, "text", "text")
+	topLevelComment4, _ := person.CommentOnPost(post, nil, "text", "text")
+	topLevelComment5, _ := person.CommentOnPost(post, nil, "text", "text")
 	var topLevelParentIDs []string
 	topLevelParentIDs = append(topLevelParentIDs, strconv.Itoa(topLevelComment1.ParentID), strconv.Itoa(topLevelComment2.ParentID), strconv.Itoa(topLevelComment3.ParentID), strconv.Itoa(topLevelComment4.ParentID), strconv.Itoa(topLevelComment5.ParentID))
 	comments = append(comments, topLevelComment1, topLevelComment2, topLevelComment3, topLevelComment4, topLevelComment5)
 	if getUniqueCommentParentIDs(comments) != "0" {
 		t.Error("Top level comments should all have the parent ID of 0")
 	}
 
-	childofComment1_1, _ := person.CommentOnPost(post, &topLevelComment1, "text", "i'm a child", "")
-	childofComment1_2, _ := person.CommentOnPost(post, &topLevelComment1, "text", "i'm a child", "")
-	childofComment1_3, _ := person.CommentOnPost(post, &topLevelComment1, "text", "i'm a child", "")
-	childofComment2_1, _ := person.CommentOnPost(post, &topLevelComment2, "text", "i'm a child", "")
-	childofComment3_1, _ := person.CommentOnPost(post, &topLevelComment3, "text", "i'm a child", "")
+	childofComment1_1, _ := person.CommentOnPost(post, &topLevelComment1, "text", "i'm a child")
+	childofComment1_2, _ := person.CommentOnPost(post, &topLevelComment1, "text", "i'm a child")
+	childofComment1_3, _ := person.CommentOnPost(post, &topLevelComment1, "text", "i'm a child")
+	childofComment2_1, _ := person.CommentOnPost(post, &topLevelComment2, "text", "i'm a child")
+	childofComment3_1, _ := person.CommentOnPost(post, &topLevelComment3, "text", "i'm a child")
```

Comment:
again, not part of the refactor, but necessary for passing tests because of the latest migration